src/status_im2/contexts/chat/composer/view.cljs

```diff
@@ -23,7 +23,21 @@
     [status-im2.contexts.chat.composer.handlers :as handler]
     [status-im2.contexts.chat.composer.gradients.view :as gradients]
     [status-im2.contexts.chat.composer.selection :as selection]
-    [quo2.theme :as theme]))
+    [quo2.theme :as theme]
+    [quo2.core :as quo]))
+
+
+(defn styled-text-input-content
+  [input-with-mentions]
+  (reduce (fn
+            [styled-text [chunk-type chunk-content]]
+            (cond
```

Comment:
```
(condp = chunk-type
   :text  (conj..
   :mention (conj ...)
```
Avoids `case` as it's not safe and it's a bit more compact than just cond

Reply:
Thanks!